namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/resources/ecr.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
```

Comment:
The 'backend' and 'provider' are also specified in your `main.tf` - this is causing terraform to fail.

Reply:
amended